app/src/components/new-request/stats.vue

```diff
@@ -125,22 +145,45 @@ export default class Stats extends Vue {
 
   /** The regular wait time, in days. */
   get regularWaitTime (): string | number {
-    const regularWaitTime = GetFeatureFlag('hardcoded_regular_wait_time')
-    if (regularWaitTime > 0) {
-      return regularWaitTime
-    } else {
-      return (this.getStats?.regular_wait_time ?? '-')
+    const flagVal = GetFeatureFlag('hardcoded_regular_wait_time')
+
+    if (flagVal > 0) return flagVal
+    if (flagVal < 0) return '-'
+
+    // when flag is 0 or invalid, fall back to stats value
```

Comment:
How could the FF be invalid? What value would that be?

Reply:
it will be 0 if FF is invalid or disabled